run_apt_packages_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# enable fail detection...
+set -e
+
+echo "StrictHostKeyChecking no" >> /etc/ssh/ssh_config
+HOSTIP=$(vagrant ssh-config | awk '/HostName/ {print $2}')
+
+echo -e "\nSETTING up Dokku SSH key.\n"
+cat /.ssh/id_rsa.pub | vagrant ssh -c "docker exec -i dokku sshcommand acl-add dokku root"
+
+echo -e "\nCLONING repo\n"
+git clone https://github.com/experimental-platform/nodejs-hello-world.git
+cd nodejs-hello-world/
+echo -e "\nCHECKING OUT branch\n"
+git checkout example_aptfile
+git config user.email "platform@protonet.info"
+git config user.name "Protonet Integration Test node.js"
+# http://progrium.viewdocs.io/dokku/checks-examples.md
+echo -e "WAIT=10\nATTEMPTS=20\n/ GraphicsMagick" > CHECKS
```

Comment:
@hinnerk I think this is the reason why the tests are failing: ``GraphicsMagick`` is only returned when the apt-get worked. So this should be fine once we have the dokku plugin in place